components/org.wso2.carbon.identity.oauth.endpoint/src/test/java/org/wso2/carbon/identity/oauth/endpoint/jwks/JwksEndpointTest.java

```diff
@@ -228,6 +221,8 @@ protected Map<String, Object> initialValue() {
                 keyStoreManager.when(() -> KeyStoreManager.getInstance(anyInt())).thenReturn(mockKeyStoreManager);
                 lenient().when(mockKeyStoreManager.getKeyStore("foo-com.jks")).thenReturn(
                         getKeyStoreFromFile("foo-com.jks", "foo.com"));
+                lenient().when(mockKeyStoreManager.getPrimaryKeyStore()).thenReturn(
```

Comment:
Why to we include file retrieval in the test ?

Reply:
Although we are moving to PKCS12 keystores, still we have JKS keystores in other repositories for unit tests